iotlabcli/experiment.py

```diff
@@ -64,14 +64,14 @@ def submit_experiment(api, name, duration,  # pylint:disable=too-many-arguments
 
     exp_files = helpers.FilesDict()
     for res_dict in resources:
-        experiment.add_exp_resources(res_dict)
         exp_files.add_files_from_dict(NODES_ASSOCIATIONS_FILE_ASSOCS, res_dict)
+        experiment.add_exp_resources(res_dict)
```

Comment:
Why are these out of the previous commit ? The commit message does not explain what you are doing here.

Reply:
add_files_from_dict modifies res_dict (because we are modifying the file name), that's why. Will update the commit message or squash